### PR TITLE
Feedback Page Added

### PR DESCRIPTION
**Pull request by sharur7**
@smaranjitghose review my PR, please. Thank You!
#18

### CODE REVIEW

File: Feedback Page/feedback.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+<html lang="en">
+
+<head>
+ 	<title>Feedback Form</title>
+ 	<meta charset="utf-8">
+ 	<meta name="viewport" content="width=device-width, initial-scale=1">
+ 	<link href="https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">		
+	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+	<link rel="stylesheet" href="Style.css">
```
Comment (smaranjitghose) [inline Feedback Page/feedback.html:11]: Bug in file name. As per your definition, it's ``style.css``

File: Feedback Page/style.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
+body{
+	background-color: #9b93f5;
+	font-family:sans-serif;
```
Comment (smaranjitghose) [inline Feedback Page/style.css:3]: Use Poppins font please

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
# Requested Changes:
- Please put the feedback page at the root level of the directory. Otherwise it would be difficult to access it
- Update the NavBar
- **Do not name the css as _style.css_. Rename it as feedback.css**
- CSS,JS and images are files are to be put inside the assets folder. Please check how it is being done for other folders and fix this
- The HTML and CSS files lack documentation. Please comment on each part elaborately

**Comment by sharur7** [comment]
> # Requested Changes:
> * Please put the feedback page at the root level of the directory. Otherwise it would be difficult to access it
> * Update the NavBar
> * **Do not name the css as _style.css_. Rename it as feedback.css**
> * CSS,JS and images are files are to be put inside the assets folder. Please check how it is being done for other folders and fix this
> * The HTML and CSS files lack documentation. Please comment on each part elaborately

Yes, I'll make the changes. I have a doubt that do I have to make changes in the index.html file to update the navigation bar or add Navbar to my feedback page?

File: Feedback Page/feedback.html (hunk 1 above)
Comment (sharur7) [reply Feedback Page/feedback.html]: done

File: Feedback Page/style.css (hunk 2 above)
Comment (sharur7) [reply Feedback Page/style.css]: done

**Review by anushbhatia — CHANGES_REQUESTED** [review]
See either https://codepen.io/josephrexme/details/oGWwOb
https://codepen.io/oliviale/pen/mgWjpq u can add these
if you are having a background which is dull make it dark and make the text light or you can add a card on the form and add a thank you form on filling that makes you any way possible.

**Pull request closed without merging**